tutorial/tuto4.py

```diff
@@ -2,11 +2,6 @@
 
 
 class PDF(FPDF):
-    def __init__(self):
```

Comment:
Good job in updating this example!

It's impressive how you simplified this demo script!